packages/react/src/tags/docs/overview.mdx

```diff
@@ -70,25 +74,34 @@ Please avoid using Tags with links and remove buttons together, as it is confusi
 		{ label: 'Orange' },
 		{ label: 'Peach' },
 	]);
+	const [announcement, setAnnouncement] = React.useState('');
 
 	const handleRemove = (label) => {
 		setItems((prevItems) => prevItems.filter((item) => item.label !== label));
+		setAnnouncement(`${label} removed`);
 	};
 
 	const itemsWithHandlers = items.map((item) => ({
 		...item,
 		onRemove: () => handleRemove(item.label),
 	}));
 
-	return items.length > 0 ? (
-		<Tags
-			heading={
-				<Text as="h2" fontWeight="bold">
-					Tags:
-				</Text>
-			}
-			items={itemsWithHandlers}
-		/>
-	) : undefined;
+	return (
+		<>
+			<Box css={visuallyHiddenStyles} role="status">
```

Comment:
Elsewhere we just used a `div` for the hidden status. Given our performance issues with `Box`, I wonder if it's worthwhile maintaining that approach here, even if it's only a minor gain.

Reply:
had to do this since I can't apply `visuallyHiddenStyles` to a `div` in the code examples

Reply:
Ultimately, I think we should make VisuallyHidden much more customisable to support role etc, so users can just do `<VisuallyHidden role="status">`